packages-ts/gauntlet-terra-contracts/src/commands/contracts/multisig/multisig.ts

```diff
@@ -0,0 +1,67 @@
+// multisig.ts
+//
+// For now, propose, vote, and execute functionality are all combined into one CONTRACT:COMMAND::multisig meta-command
+//  This is parallel to how things are implemented in Solana.  The execute happens automatically as soon as the last
+//  vote required to exceeed the threshold is cast.  And the difference between propose and vote is distinguished by
+//  whether the --proposal=PROPOSAL_HASH flag is passed.  Later, we may want to split this into CONTRACT::COMMAND::propose,
```

Comment:
We want to stick to the same process as EVM and Solana, where the multisig command tells us which step we are based on the proposal state